src/Source.ml

```diff
@@ -23,6 +23,11 @@ let string_between t (l1 : Location.t) (l2 : Location.t) =
     (* can happen e.g. if comment is within a parenthesized expression *)
     None
 
+let empty_line_between t l1 l2 =
+  let is_empty s = String.strip s |> String.is_empty in
```

Comment:
Perhaps premature optimization, but I'd be tempted to avoid copying the string by using `String.lfindi` with an `~f` that checks for non-whitespace chars.